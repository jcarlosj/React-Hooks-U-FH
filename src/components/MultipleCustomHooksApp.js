import React from 'react';

export const MultipleCustomHooksApp = () => {
    return (
        <section className="container mt-5">
            <h1>Custom Hooks</h1>  
            <hr /> 
        </section>
    )
}
