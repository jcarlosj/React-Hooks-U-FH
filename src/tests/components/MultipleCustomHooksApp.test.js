import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import { MultipleCustomHooksApp } from '../../components/MultipleCustomHooksApp';

describe( '<MultipleCustomHooksApp />', () => {

    test( 'debe desplegar correctamente', () => {

        const wrapper = shallow( 
            <MultipleCustomHooksApp /> 
        );
        
        expect( wrapper ).toMatchSnapshot();

    } );

} );