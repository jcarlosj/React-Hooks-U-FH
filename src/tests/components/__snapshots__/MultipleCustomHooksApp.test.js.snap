// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultipleCustomHooksApp /> debe desplegar correctamente 1`] = `
<section
  className="container mt-5"
>
  <h1>
    Breaking Bad Quotes
  </h1>
  <p>
    Custom Hooks
  </p>
  <hr />
  <p
    className="alert alert-info text-center"
  >
    Loading...
  </p>
</section>
`;
